superbench/benchmarks/result.py

```diff
@@ -64,9 +65,22 @@ def add_raw_data(self, metric, value):
             )
             return False
 
-        if metric not in self.__raw_data:
-            self.__raw_data[metric] = list()
-        self.__raw_data[metric].append(value)
+        export_raw_data = 1
```

Comment:
Why do we need to init this? If the environment doesn't work, we will error directly

Reply:
I think this environment needn't to break the whole execution, so just log the error message and use the default value.

Reply:
removed.